test/Interop/SwiftToCxx/class/nested-classes-in-cxx.swift

```diff
@@ -0,0 +1,31 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend %s -enable-library-evolution -typecheck -module-name Classes -clang-header-expose-decls=all-public -emit-clang-header-path %t/classes.h
+// RN: %FileCheck %s < %t/structs.h
```

Comment:
Is this a typo?

```suggestion
// RUN: %FileCheck %s < %t/classes.h
```

And I don't see any `// CHECK` statements in this file. What is this supposed to do?

I was also going to ask whether there's any way to check the textual contents of the C++ header file generated from this Swift code (similar to the `-module-interface.swift` files for many forward interop test cases), but I suspect that's what you mean to do here.